src/app/models/transmart-models/transmart-studies-dimensions.ts

```diff
@@ -0,0 +1,8 @@
+import {Dimension} from "../table-models/dimension";
+import {TransmartTableState} from "app/models/transmart-models/transmart-table-state";
+
+export class TransmartStudiesDimensions {
+  availableDimensions: Array<Dimension> = [];
+  defaultTableRepresentation: TransmartTableState;
```

Comment:
'defaultTableRepresentation: TransmartTableState' is confusing when compared to 'defaultTabularRepresentation: TransmartDefaultTableRepresentation'